tests/everflow/test_everflow_testbed.py

```diff
@@ -87,6 +87,22 @@ class EverflowIPv4Tests(BaseEverflowTest):
     DEFAULT_DST_IP = "30.0.0.1"
     MIRROR_POLICER_UNSUPPORTED_ASIC_LIST = ["th3", "j2c+", "jr2"]
 
+    @staticmethod
+    def _is_ipv6_only_topology(tbinfo):
+        """Helper function to determine if this is an IPv6 topology."""
+        return (
+            "-v6-" in tbinfo["topo"]["name"]
+            if tbinfo and "topo" in tbinfo and "name" in tbinfo["topo"]
+            else False
+        )
+
+    @pytest.fixture(autouse=True)
```

Comment:
The fixture can be class level

Reply:
Yes, lets let this one in. Since many place used the same check, I will push a PR to fix them all to use the same function.

Reply:
Going to change to call the is_ipv6_only_topology in #19639